packages/react/src/components/Combobox/Combobox.tsx

```diff
@@ -280,10 +401,17 @@ const Combobox = forwardRef<HTMLDivElement, ComboboxProps>(
 
         setOpen(true);
 
-        if (!open && arrowKeypress && selectedValue && isAutoComplete) {
+        if (
+          inputValue === '' &&
+          (backspaceKeypress || deleteKeypress || event.key === 'ArrowLeft')
+        ) {
+          pillsRef.current[pillsRef.current.length - 1]?.focus();
```

Comment:
If we're using an array ref, we need to guard the ref by either:

- Ensuring we can't get an out of range value
- Only invoking `focus` when there's actually selected items

It looks like it might be possible to go negative here.

Reply:
even if it will be negative, result of 

```
const array = [];
console.log(array[-1]);
```

<img width="382" alt="image" src="https://github.com/user-attachments/assets/7ef04fb0-c84f-4c5d-9709-59a096161bbf" />

Will be undefined

Reply:
I still think it would be good to avoid this logic block entirely if we knew there weren't any pills present.